UPGRADING.md

```diff
@@ -130,5 +116,23 @@ mistakes.
 
 ### Modules
 
+#### `x/params`
+
 * The `x/param` module has been depreacted in favour of each module housing and providing way to modify their parameters. Each module that has parameters that are changable during runtime have an authority, the authority can be a module or user account. The Cosmos-SDK team recommends migrating modules away from using the param module. An example of how this could look like can be found [here](https://github.com/cosmos/cosmos-sdk/pull/12363). 
-    * The Param module will be maintained until April 18, 2022. At this point the module will reach end of life and be removed from the Cosmos SDK.
+* The Param module will be maintained until April 18, 2023. At this point the module will reach end of life and be removed from the Cosmos SDK.
+
+#### `x/gov`
+
+The `gov` module has been greatly improved. The previous API has been moved to `v1beta1` while the new implementation is called `v1`.
+
+In order to submit a proposal with `submit-proposal` you now need to pass a `proposal.json` file.
+You can still use the old way by using `submit-legacy-proposal`. This is not recommended.
+More information can be found in the gov module [client documentation](https://docs.cosmos.network/v0.46/modules/gov/07_client.html).
+
+### Protobuf
+
+The `third_party/proto` folder that existed in [previous version](https://github.com/cosmos/cosmos-sdk/tree/v0.45.3/third_party/proto) now does not contains directly the [proto files](https://github.com/cosmos/cosmos-sdk/tree/release/v0.46.x/third_party/proto).
+
+Instead, the SDK uses [`buf`](https://buf.build). Clients should have their own [`buf.yaml`](https://docs.buf.build/configuration/v1/buf-yaml) with `buf.build/cosmos/cosmos-sdk` as dependency, in order to avoid having to copy paste these files.
+
+The protos can as well be downloaded using `buf export buf.build/cosmos/cosmos-sdk:$(git log -1 --pretty=format:"%H" v0.46.0) --output <some_folder>`.
```

Comment:
I seems like this is not working. The commit for v0.46.0 is `a1143138716b64bc4fa0aa53c0f0fa59eb675bb7`:

```
buf export buf.build/cosmos/cosmos-sdk:a1143138716b64bc4fa0aa53c0f0fa59eb675bb7 --output .
Failure: buf.build/cosmos/cosmos-sdk:a1143138716b64bc4fa0aa53c0f0fa59eb675bb7: does not exist
```

Reply:
Right, right now, the buf action only push at proto changes. https://github.com/cosmos/cosmos-sdk/blob/main/.github/workflows/proto-registry.yml.

I am trying to see if it is possible to as well push it at tag creation, which is a better ux than getting the kast commit that contained a proto change before the release.

Reply:
Ideally we had the tag on the buf registry directly. But I don't know if this is possible.

Reply:
Okay, it should work now :)
I do hope this ever gets implemented: https://github.com/bufbuild/buf/issues/765

Reply:
Beautiful!

Now I suggest documenting this instead, because it does not require a client to be operating in a cosmos-sdk checkout:

```
buf export buf.build/cosmos/cosmos-sdk:$(curl -sS https://api.github.com/repos/cosmos/cosmos-sdk/commits/v0.46.0 | jq -r .sha) --output <some_folder>
```